tests/unit/source_code/linters/test_pyspark.py

```diff
@@ -1,22 +1,23 @@
-import ast
-
 import pytest
 
+from astroid import Call, Const, Expr  # type: ignore
+
 from databricks.labs.ucx.source_code.base import Deprecation, CurrentSessionState
-from databricks.labs.ucx.source_code.linters.pyspark import SparkSql, AstHelper, TableNameMatcher
+from databricks.labs.ucx.source_code.linters.imports import ASTLinter
+from databricks.labs.ucx.source_code.linters.pyspark import AstHelper, TableNameMatcher, SparkSqlLinter
 from databricks.labs.ucx.source_code.queries import FromTable
 
 
 def test_spark_no_sql(empty_index):
     ftf = FromTable(empty_index, CurrentSessionState())
-    sqf = SparkSql(ftf, empty_index)
+    sqf = SparkSqlLinter(ftf, empty_index)
```

Comment:
please revert this rename, it bloats the diff

Reply:
Fixed